geomesa-spark/geomesa-spark-jts/pom.xml

```diff
@@ -41,6 +41,18 @@
             <groupId>org.locationtech.spatial4j</groupId>
             <artifactId>spatial4j</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-shapefile</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-epsg-hsql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-opengis</artifactId>
```

Comment:
Sorry if I wasn't clear previously in Gitter.  This is what @elahrvivaz was mentioning in another comment.  This module will not include GeoTools (to allow license decoupling).  Admittedly, this is a somewhat arbitrary distinction.